packages/storybook/src/nimble/text-field/text-field.stories.ts

```diff
@@ -65,31 +68,77 @@ const metadata: Meta<TextFieldArgs> = {
         </${textFieldTag}>
     `),
     argTypes: {
+        label: {
+            name: 'default',
+            description: `${slottedLabelDescription({ componentName: 'text field' })}`,
+            table: { category: apiCategory.slots }
+        },
+        placeholder: {
+            description: placeholderDescription({ componentName: 'text field' }),
+            table: { category: apiCategory.attributes }
+        },
         type: {
             options: Object.values(TextFieldType),
-            control: { type: 'select' }
+            control: { type: 'select' },
+            description: 'They type of input to accept and render in the text field. This corresponds to [the `type` attribute of the native `input` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#type) though only a subset of values are supported.',
```

Comment:
Shouldn't need to open the dropdown to see what values are supported. Alternatively, could use radio buttons instead.
```suggestion
            description: `The type of input to accept and render in the text field. This corresponds to [the \`type\` attribute of the native \`input\` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#type) though only the following values are supported:
   - text (default)
   - email
   - password
   - tel
   - url`,
```

Reply:
Changed it to a radio button.